src/Backend/InferenceServer/inference.py

```diff
@@ -29,12 +30,14 @@ def infer_image(word_model: YOLO, img_array):
     img = cv2.imdecode(img_array, cv2.IMREAD_COLOR)
     img = cv2.cvtColor(img, cv2.COLOR_RGB2BGR)
     img = cv2.cvtColor(img, cv2.COLOR_BGR2GRAY)
-    img = highPassFilter(img)
-    img = whitePointSelect(img)
-    img = blackPointSelect(img)
-    _, img = cv2.threshold(img, 0, 255, cv2.THRESH_BINARY + cv2.THRESH_OTSU)
-    img = rotate(img, determine_skew(img),
-                 border_mode=cv2.BORDER_CONSTANT, border_value=255)
+    #img = highPassFilter(img)
+    #img = whitePointSelect(img)
+    #img = blackPointSelect(img)
+    # https://github.com/scikit-image/scikit-image/blob/main/doc/examples/segmentation/plot_niblack_sauvola.py
+    # using defaults
+    img = threshold_sauvola(img, window_size=25)
```

Comment:
Per @ramezgerges request.

```suggestion
    img = threshold_sauvola(img, window_size=15)
```